File: service/native-java-quarkus/src/main/java/com/ecobenchmark/entities/Task.java
package com.ecobenchmark.entities;

import com.ecobenchmark.controllers.getstats.StatsResponse;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;
import java.time.Instant;
import java.util.UUID;

@SqlResultSetMapping(
        name = "StatsResponseMapping",
        classes = {
                @ConstructorResult(
                        targetClass = StatsResponse.class,
                        columns = {
                                @ColumnResult(name = "id" ,type = UUID.class),
                                @ColumnResult(name = "login", type = String.class),
                                @ColumnResult(name = "nb_list", type = Integer.class),
                                @ColumnResult(name = "avg_tasks", type = Float.class)})})
@Entity
public class Task {

    @Id
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    private UUID id;

    private String name;

    private String description;

    @Column(name = "creation_date")
    private Instant creationDate;

    @ManyToOne
    private ListEntity list;

    public Task(String name, String description, Instant creationDate, ListEntity list) {
        this.name = name;
        this.description = description;
        this.creationDate = creationDate;
        this.list = list;
    }

    public Task() {
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Instant getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(Instant creationDate) {
        this.creationDate = creationDate;
    }

    public ListEntity getList() {
        return list;
    }

    public void setList(ListEntity list) {
        this.list = list;
    }
}

File: service/php-symfony-apache2-dev/src/Entity/Task.php
<?php

namespace App\Entity;

use App\Repository\TaskRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Uid\UuidV4;

#[ORM\Entity(repositoryClass: TaskRepository::class)]
class Task
{
    #[ORM\Id]
    #[ORM\Column(type: 'uuid', unique: true)]
    #[ORM\GeneratedValue(strategy: "CUSTOM")]
    #[ORM\CustomIdGenerator(class: 'doctrine.uuid_generator')]
    private $id;

    #[ORM\ManyToOne(targetEntity: ListEntity::class, inversedBy: 'tasks')]
    private $list;

    #[ORM\Column(type: 'text', nullable: true)]
    private $name;

    #[ORM\Column(type: 'text')]
    private $description;

    #[ORM\Column(type: 'datetimetz', nullable: true)]
    private $creation_date;

    public function getId(): ?UuidV4
    {
        return $this->id;
    }

    public function getList(): ?ListEntity
    {
        return $this->list;
    }

    public function setList(?ListEntity $list): self
    {
        $this->list = $list;

        return $this;
    }

    public function getName(): ?string
    {
        return $this->name;
    }

    public function setName(?string $name): self
    {
        $this->name = $name;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getCreationDate(): ?\DateTimeInterface
    {
        return $this->creation_date;
    }

    public function setCreationDate(?\DateTimeInterface $creation_date): self
    {
        $this->creation_date = $creation_date;

        return $this;
    }
}
